cmd/indexer/root.go

```diff
@@ -180,9 +175,12 @@ func init() {
 		ServerCmd.Flags().Int64Var(&fromBlock, flags.SyncFromBlock, 0, "The init block number to sync to initially")
 
 		// Profling flags
-		ServerCmd.Flags().BoolVar(&profiling, "pprof", false, "Enable the pprof HTTP server")
-		ServerCmd.Flags().IntVar(&profilPort, "pprof.port", 8000, "pprof HTTP server listening port")
-		ServerCmd.Flags().StringVar(&profilHost, "pprof.host", "0.0.0.0", "pprof HTTP server listening interface")
+		ServerCmd.Flags().BoolVar(&profiling, flags.PprofEnable, false, "Enable the pprof HTTP server")
+		ServerCmd.Flags().IntVar(&profilPort, flags.PprofPort, 8000, "pprof HTTP server listening port")
+		ServerCmd.Flags().StringVar(&profilHost, flags.PprofHost, "0.0.0.0", "pprof HTTP server listening interface")
+
+		// erc20 flags
+		ServerCmd.Flags().BoolVar(&subscribeErc20token, flags.SubscribeErc20token, false, "Enable subscribe erc20 token")
```

Comment:
can we add some words in the description as well?
```
Enable erc20 token subscription. Please specify the erc20 tokens in configs/erc20.yaml
```

Reply:
OK